ale/base/data_isis.py

```diff
@@ -836,37 +649,4 @@ def odtk(self):
         """
         return self.label["NaifKeywords"]["INS{}_OD_K".format(self.ikid)]
```

Comment:
What happens if the label does not have the `NaifKeywords` key? The user will see a weird stack trace I suspect. Does it makes sense to do a more explicit check for that key, throw an informative error if it does not exist ad then dig in to the key of interest?